server/game/CardAbility.js

```diff
@@ -79,7 +79,7 @@ class CardAbility extends ThenAbility {
             this.card.owner.registerAbilityMax(this.maxIdentifier, this.max);
         }
 
-        if(card.getType() === CardTypes.Event) {
+        if(card.getType() === CardTypes.Event && !this.isKeywordAbility()) {
```

Comment:
We might want to find some other way to discriminate here. Even though we don't have to implement the Support keyword right now, it could become relevant at any time.

Reply:
I think if we implement support, it would be via a constant effect that we check in payReduceableFateCost and not as a triggered keyword ability

Reply:
Fair point. I'm failing to come up with a hypothetical for a keyword on an event that would play the card when triggered.